test/src/sdk-harness/test_artifacts/storage_vec/svec_bool/Cargo.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "svec_bool"
+version = "0.1.0"
+authors = ["Swaystar"]
```

Replace any other instances too.
```suggestion
authors = ["Fuel Labs <contact@fuel.sh>"]
```